content/en/docs/Reference/catalog-templates.md

```diff
@@ -503,3 +503,357 @@ schema: olm.channel
+      config:
+        input: components/v4.11.yaml 
+        output: catalog.yaml
+- name: v4.12
+  destination:
+    path: my-package
+  strategy:
+    name: raw
+    template:
+      schema: olm.builder.raw {{< code_callout 7 >}}
+      config:
+        input: components/v4.12.yaml 
+        output: catalog.yaml
+- name: v4.13
+  destination:
+    path: my-package
+  strategy:
+    name: custom
+    template:
+      schema: olm.builder.custom {{< code_callout 8 >}}
+      config:
+        command: cat
+        args:
+          - "components/v4.13.yaml"
+        output: catalog.yaml
+```
+
+- {{< code_callout 1 >}} Defines the build process for the `v4.10` catalog contribution. All catalogs defined in the composite template configuration file **must** also exist in the catalog configuration file. If the composite configuration includes a component that is not defined in the catalog configuration, the composite template will fail to render 
+- {{< code_callout 2 >}} Defines the name of the package being contributed and is used to create the package directory
+- {{< code_callout 3 >}} Specifies that the contribution for the `v4.10` catalog is a basic template so that the contribution can be rendered appropriately
+- {{< code_callout 4 >}} Specifies the basic template file that should be used to render into FBC
+- {{< code_callout 5 >}} Defines what the name of the output file should be. This is joined as a path with the  `destination.path` value defined in {{< code_callout 2 >}}. In this example the full output path after rendering would be `my-package/catalog.yaml`
+- {{< code_callout 6 >}} Specifies the contribution for the `v4.11` catalog is a semver template
+- {{< code_callout 7 >}} Specifies the contribution for the `v4.12` catalog is already full FBC
+- {{< code_callout 8 >}} Specifies the contribution for the `v4.13` catalog requires running custom commands that result in a full FBC being printed to STDOUT. In this example we are just running the `cat` command on a file that is full FBC
+
+
+Using the above examples, the `components/` directory should look similar to:
+```tree
+components
+├── v4.10.yaml
+├── v4.11.yaml
+├── v4.12.yaml
+└── v4.13.yaml
+```
+
+For more details about the contents of each file, expand the corresponding dropdown:
+<details>
+<summary>components/v4.10.yaml</summary>
+
+```yaml
+---
+defaultChannel: preview
+name: webhook-operator
+schema: olm.package
+---
+image: quay.io/olmtest/webhook-operator-bundle:0.0.3
+name: webhook-operator.v0.0.1
+schema: olm.bundle
+---
+schema: olm.channel
+package: webhook-operator
+name: preview
+entries:
+  - name: webhook-operator.v0.0.1
+```
+
+</details>
+
+<details>
+<summary>components/v4.11.yaml</summary>
+
+```yaml
+Schema: olm.semver
+GenerateMajorChannels: true
+GenerateMinorChannels: true
+Stable:
+  Bundles:
+  - Image: quay.io/olmtest/webhook-operator-bundle:0.0.3
+```
+
+</details>
+
+<details>
+<summary>components/v4.12.yaml</summary>
```

Comment:
I'm not convinced that we need to go to the level of detail in the example of providing all the content of all the participants.  On the other hand, I can see us donating the example repo to OF and making a reference at the top of the example that we're referring to LOCATION for the example. 

If we do find all this detail useful, I think it's sufficient if we strip out all `olm.bundle.object` objects and make a note that we're filtering them out (similar to examples for other template types on this page).